src/entities/test-summary-review-and-submit.entity.ts

```diff
@@ -0,0 +1,71 @@
+import { ViewColumn, ViewEntity } from 'typeorm';
```

Comment:
let's stay consistent with the code properties and spell out the word "Code"
orisCode
testTypeCode
evalStatusCode
submissionAvailabilityCode